### DISCUSSION TITLE
I want to be able to compare two specific change requests

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#127] [Feature Requests]
### What's your feature/idea?

I want to be able to pick any two change requests from history and compare them for changes. 
Something like what Confluence offers. 

### What's your desired outcome?

Be able to compare two specific change requests in gitbook UI to compare what exactly changed between the two.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by denisemcg** [comment]
I need this too. Sometimes I need to remind myself of what used to be or revert to older content. Plus when someone else makes a change, I want to quickly see what's different. This is particularly important for long pages. It's not much good telling us a page has changed without any indicator of what was actually changed. Confluence does this well...
![image](https://github.com/user-attachments/assets/197a9268-a959-4476-a3e9-c85f3ec8407f)
